src/components/progressBar/progressBar.style.js

```diff
@@ -0,0 +1,33 @@
+import {color} from '../../theme/style';
+import {size} from '../../theme/size';
```

Comment:
the same. import from index.

Reply:
solved the issue, it was the naming of the folder